### PR TITLE
Adds docs and corrects type-o

### PR DESCRIPTION
**Pull request by yunghuffy**
Short description explaining the high-level reason for the pull request

## Additions
Documentation
## Removals

None
## Changes

All text
## Testing

1. Proofreading

## Review

- @acrewdson @OrlandoSoto 

[Preview this PR without the whitespace changes](?w=0)

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,111 @@
-# CFPB Jenkins pipeline shared libraries
+# CFPB Jenkins Pipeline Shared Libraries
 
-TODO description tk
+This is a collecitons of shared libraries to be used with
```
Comment (acrewdson) [inline README.md]: 'a collection'

File: README.md [hunk 2]
```diff
@@ -1,40 +1,111 @@
...
+Jenkins >= 2.46.3, though some earlier versions may work, they have not been
+tested.
 
-## Getting help
+## Installation
 
-tk
+These shared libraries may be accessed one of three ways:
+1. Add this repo to 'Global Pipeline Libraries' in the Jenkins UI.
+1. Include a `libraries` blocking in declarative pipeline syntax.
```
Comment (acrewdson) [inline README.md]: 'block'

File: README.md [hunk 3]
```diff
@@ -1,40 +1,111 @@
...
+Since this repo is hosted on github.com, it's easy to include it in all
+script dynamically by adding it to the `@Library` block within a Pipeline script
+
+```
+@Library('github.com/cfpb/jenkins-pipeline-shared-libraries') _
+```
+
+## Usage
+
+Currently this repo has a `sendEmail` var that can be inovked after
```
Comment (acrewdson) [inline README.md]: 'invoked'

File: README.md [hunk 4]
```diff
@@ -1,40 +1,111 @@
...
+importing it. Once the entire share library has been imported, `sendEmail`
```
Comment (acrewdson) [inline README.md]: 'shared'

File: README.md [hunk 5]
```diff
@@ -1,40 +1,111 @@
...
+
+## Getting help
+
+If you have any trouble working with this repo, please open a github issue.
 
-General instructions on _how_ to contribute should be stated with a link to [CONTRIBUTING](CONTRIBUTING.md).
+## Getting involved
+
+Adding libraries to this repo is welcome and encouraged. Should you have
+an idea for a useful library or an improvement for an existing one fork
```
Comment (acrewdson) [inline README.md:91]: 'to an'

**Review by acrewdson — CHANGES_REQUESTED** [review]
couple typos, otherwise LGTM

File: README.md (hunk 5 above)
Comment (yunghuffy) [reply README.md:91]: This is not clear. Can you provide more context?

Comment (acrewdson) [reply README.md:91]: 'improvement to an' is more idiomatic than 'improvement for an' IMO, but please go with the one that sounds better to you

Comment (yunghuffy) [reply README.md:91]: I'm content with 'for an'. Let's stick with it.

**Review by acrewdson — APPROVED** [review]

**Pull request merged**